content/webapp/views/pages/concepts/concept/concept.ImagesResults.tsx

```diff
@@ -78,7 +81,7 @@ const ImageSection: FunctionComponent<Props> = ({
       data-id={`images-${type}`}
     >
       <SectionHeading id={`images-${type}`}>
-        Images {getThemeSectionHeading(type, concept, true)}
+        {getSectionTypeLabel(type, config, 'images')}
```

Comment:
Because this can return undefined, should we have a fallback (everywhere it's used)?

Reply:
Good idea yes!